resources/fieldsets/text_columns.yaml

```diff
@@ -0,0 +1,120 @@
+title: 'Text columns'
+fields:
+  -
+    import: component_settings
+  -
+    handle: title_heading
+    field:
+      input_type: text
+      antlers: false
+      type: text
+      display: 'Title - Heading'
+      icon: text
+      listable: hidden
+      instructions_position: above
+      visibility: visible
+      replicator_preview: true
+      hide_display: false
+      width: 50
+  -
+    handle: title_subheading
+    field:
+      input_type: text
+      antlers: false
+      type: text
+      display: 'Title - Subheading'
+      icon: text
+      listable: hidden
+      instructions_position: above
+      visibility: visible
+      replicator_preview: true
+      hide_display: false
+      width: 50
+  -
+    handle: columns
+    field:
+      collapse: true
+      previews: true
+      fullscreen: true
+      sets:
+        extra_column:
+          display: 'Extra column'
+          instructions: null
+          icon: null
+          sets:
+            new_set:
+              display: Column
+              instructions: null
+              icon: null
+              fields:
+                -
+                  handle: title
+                  field:
+                    input_type: text
+                    antlers: false
+                    type: text
+                    display: Title
+                    icon: text
+                    listable: hidden
+                    instructions_position: above
+                    visibility: visible
+                    replicator_preview: true
+                    hide_display: false
+                -
+                  handle: text
```

Comment:
I see this field a lot as well, if we want the same configuration like buttons and all the other stuff, why no fieldset if we ever want to change it?

Reply:
Agreed, i'd like to see a `text` fieldset apart from the `content` fieldset that already exists.